.gitattributes

```diff
@@ -11,6 +11,8 @@
 *.pem eol=lf
 
 # Denote all files that are truly binary and should not be modified.
+*.avif binary
+*.webp binary
 *.png binary
```

Comment:
The core doesn't these type of files so there isn't much value in this change. We can instead add avif to the .gitattributes of cakephp/app repo (it already has webp).

Reply:
>  Do you want i make the push on app ?

Yes please, that is how we push updates to the application skeleton code. New users generally use the latest app skeleton and existing applications can overlay the new files on theirs to selectively apply changes in the default skeleton.